src/index.tsx

```diff
@@ -9,6 +9,11 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import "./index.scss";
 import reportWebVitals from "./reportWebVitals";
+import { makeServer } from "./server";
+
+if (process.env.NODE_ENV === "development") {
```

Comment:
Update readme to reflect use of mirage and how you can toggle it in dev so you can use a real API. Update package.json to reflect that toggle.